src/components/ReviewListCard.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+
+export default function ReviewListCard({ review }) {
+  return (
+    <Card sx={{ minWidth: 275 }}>
+      <CardContent>
+        <h3>{review.title}</h3>
+        <img
+          className="review-img"
+          src={review.review_img_url}
+          alt="The reviewed game"
+        />
+        <br />
+        <p>
+          <div className="review-list-footer">
+            <p className="review-list-category">Category: {review.category}</p>
```

Comment:
React is complaining vehemently about these `<p>` tags being nested inside the `<p>` tags open on line 16. 
Suggest removing 'parent' `<p>` altogether